calculator.c

```diff
@@ -0,0 +1,32 @@
+#include <stdio.h>
```

Comment:
Great job! You code compiled and ran appropriately! The only thing I'd add is a comment at the top with your name and the function of this program - it's good practice.